api/src/test/java/org/apache/iceberg/types/TestSerializableTypes.java

```diff
@@ -112,13 +113,13 @@ public void testMaps() throws Exception {
 
   @Test
   public void testLists() throws Exception {
-    Type[] maps =
+    Type[] lists =
```

Comment:
Pull this change out into a tiny PR and I will merge it instantly :)

Reply:
Sounds good. Let me come up with a separate PR.